test.js

```diff
@@ -0,0 +1,6 @@
+console.log('foo');
+console.log('foo')
+    console.log('foo')
```

Comment:
__[staging][eslint]__
* Error - Expected indentation of 0 spaces but found 4. (indent)

<!-- Comment by [SideCI](https://sideci.com) -->